.gitignore

```diff
@@ -2,4 +2,4 @@
 roblox.yml
 
 # Build Files
-Adonis_Nightly_**.rbxm
\ No newline at end of file
+Adonis_**.rbxm
```

Comment:
i do not agree with this action

Reply:
🤖 Failed to detected problem. https://github.com/search?q=repo%3AEpix-Incorporated%2FAdonis+path%3AAdonis_**.rbxm&type=code